Sources/OuterCore/IRVerifier.swift

```diff
@@ -127,35 +127,7 @@ public final class IRVerifier {
 
   func verify(_ continuation: Continuation) -> Bool {
     return withScope {
-      var allValid = true
-      for parameter in continuation.parameters {
-        currentScopedValues.insert(parameter)
-        allValid &= verifyType(parameter.type)
-      }
-      guard let call = continuation.call else {
-        engine.diagnose(.continuationHasNoCall(continuation))
-        return false
-      }
-      guard let fnType = call.callee.type as? FunctionType else {
-        engine.diagnose(.callingNonFunction(call.callee.type))
-        return false
-      }
-
-      guard fnType.arguments.count == call.args.count else {
-        engine.diagnose(.arityMismatch(function: call.callee,
-                                       expected: fnType.arguments.count,
-                                       got: call.args.count))
-        return false
-      }
-      for (arg, param) in zip(call.args, fnType.arguments) {
-        allValid &= verifyType(arg.type)
-        allValid &= verifyType(param)
-        guard arg.type === param else {
-          engine.diagnose(.typeMismatch(expected: param, got: arg.type))
-          return false
-        }
-      }
-      return allValid
```

Comment:
Thorin verifies at the schedule-level. You could repurpose this to handle that.  Otherwise just remove it for now.